test/core/client_channel/lb_policy/xds_override_host_test.cc

```diff
@@ -112,9 +112,11 @@ class XdsOverrideHostTest : public LoadBalancingPolicyTest {
     EXPECT_EQ(ApplyUpdate(update, policy_.get()), absl::OkStatus());
   }
 
-  CallAttributes MakeOverrideHostAttribute(absl::string_view host) {
+  LoadBalancingPolicyTest::CallAttributes MakeOverrideHostAttribute(
```

Comment:
I don't think you need the `LoadBalancingPolicyTest::` prefix here, since this class is derived from that class.